openslides_backend/shared/patterns.py

```diff
@@ -32,9 +32,10 @@ def __repr__(self) -> str:
         return f"Collection({repr(str(self))})"
 
     def __eq__(self, other: object) -> bool:
-        if not isinstance(other, Collection):
-            return NotImplemented
-        return self.collection == other.collection
+        try:
+            return self.collection == other.collection  # type: ignore
```

Comment:
rather use `cast()` instead of `# type: ignore` - this should make no difference runtime-wise

Reply:
Done